component-api/src/main/java/org/talend/sdk/component/api/service/BaseService.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (C) 2006-2021 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.api.service;
+
+import java.io.ObjectStreamException;
+import java.io.Serializable;
+
+import org.talend.sdk.component.api.service.serialization.Serial;
+
+import lombok.Data;
+
+/**
+ * Classe base for services to be declared Serializable and use Component Manager serialization system.
+ */
+@Data
+public class BaseService implements Serializable {
```

Comment:
As far as I understand, we may break our connectors code base? Should we inherit from that class ?

Reply:
hum, read rest of code no breaks apparently... ;-)

Reply:
right, no breaks :-)